primitives/runtime/src/lib.rs

```diff
@@ -619,18 +620,25 @@ pub enum TokenError {
 	Frozen,
 	/// Operation is not supported by the asset.
 	Unsupported,
+	/// Account cannot be created for a held balance.
+	CannotCreateHold,
+	/// Withdrawal would cause unwanted loss of account.
+	UnwantedRemoval,
```

Comment:
This was a little confusing to me when I saw it in the code above. Would something like `UnwantedAccountRemoval` work as a clearer name? It wasn't clear without the docs that the error was refering to the account.